docker/Dockerfile.build

```diff
@@ -0,0 +1,64 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
As a general rule, I prefer to set variables in their own layer for clarity.  In this case in particular, it helps since it let me split the basic depends from the ones that are needed after the checkout.

That said, if you want I can always add it to both of the apt steps.